pkg/operator/hive/hive.go

```diff
@@ -192,9 +193,11 @@ func (r *ReconcileHiveConfig) includeAdditionalCAs(hLog log.FieldLogger, h *reso
 		return err
 	}
 	hLog.Infof("additional cert secret applied (%s)", result)
+	hash := fmt.Sprintf("%x", md5.Sum(additionalCA.Bytes()))
```

Comment:
Might be good to comment here that using the hash in the volume name ensures we see a change to the deployment when the CAs change, triggering a pod restart.